docs/quick_start.rst

```diff
@@ -98,3 +98,74 @@ error-corrected implementation of a CNOT gate shown at the beginning of this pag
 
 You can download the circuit in a ``stim`` format here:
 :download:`media/quick_start/logical_cnot.stim <./media/quick_start/logical_cnot.stim>`.
+
+6. Simulate multiple experiments
+--------------------------------
+The circuit can be simulated using the ``stim`` and ``sinter`` libraries.
+Usually you want to simulate combinations of error rates and code distances, potentially
+for multiple observables.
+Multiple runs can be done in parallel using the ``sinter`` library using the ``start_simulation_using_sinter``.
+The compilation of the block graph is done automatically based on the inputs.
+
+.. code-block:: python
+
+    import numpy as np
+
+    from tqec.noise_models import NoiseModel
+    from tqec.simulation.simulation import start_simulation_using_sinter
+
+    stats = start_simulation_using_sinter(
+        block_graph,
+        ks=range(1, 4), # k values for the code distance
+        ps=list(np.logspace(-4, -1, 10)), # error rates
+        noise_model_factory=NoiseModel.uniform_depolarizing # noise model
+        observables=[observables[1]], # observable of interest
+        decoders=["pymatching"],
+        num_workers=20,
+        max_shots=10_000_000,
+        max_errors=5_000,
+        print_progress=True,
+    )
+
+``Sinter`` can be additionally supplied with simulation parameters see
+`sinter_api https://github.com/quantumlib/Stim/blob/main/doc/sinter_api.md`_
+
+
+7. Plot the results
+-------------------
+Simulation Results can be plotted with ``matplolib`` using the ``plot_simulation_results``.
+
+.. code-block:: python
+
+    from tqec.simulation.plotting import plot_simulation_results
+
+    zx_graph = block_graph.to_zx_graph()
+
+    for i, stat in enumerate(stats):
+        fig, ax = plt.subplots()
+        sinter.plot_error_rate(
+            ax=ax,
+            stats=stat,
+            x_func=lambda stat: stat.json_metadata["p"],
+            group_func=lambda stat: stat.json_metadata["d"],
+        )
+        plot_observable_as_inset(ax, zx_graph, correlation_surfaces[i])
```

Comment:
Here, `correlation_surfaces[i]` is not really correct. `start_simulation_using_sinter` will return a list with `len(observables)` entries, `observables` being the list of observables provided to the function. Because in code cell before you only included `observables=[observables[1]]`, `stats` will only include one entry, corresponding to `observables[1]`, and so corresponding to `correlation_surfaces[1]` and not `correlation_surfaces[0]`.